agent/action/pipeline_manager.py

```
@@ -448,6 +461,52 @@ def run(self, context: Context, argv: CustomAction.RunArg) -> bool:
+        if missing_nodes:
+            utils.mfaalog.warning(f"[Py] ⚠️ RestoreBatch: 账本中未找到以下节点的备份 (忽略): {missing_nodes}")
+
+        if not restore_dict:
+            utils.mfaalog.warning("[Py] ⚠️ RestoreBatch: 没有可还原的有效节点")
+            return True
+
+        try:
+            # 4. 执行批量还原
+            context.override_pipeline(restore_dict)
+            utils.mfaalog.info(f"[Py] 🔙 批量还原成功: {list(restore_dict.keys())}")
+            return True
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)